test_package/conanfile.py

```diff
@@ -41,7 +40,5 @@ def test(self):
                 out = proc.communicate()[0]
                 pass
 
-            ConanOutput("test package").writeln(out)
```

Comment:
if i understand correctly, this just prints the output from the test command `subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, text=True)`, why do no longer want to see the output?